Small Projects/Python/Black Jack/run.py

```diff
@@ -1,186 +1,233 @@
 import random
 import os
 
-
 money = 1000
 
+
 def pick_card():
-    # Choose a random index
     random_index = random.randrange(len(list_card))
-
-    # Get the random number
-    random_number = list_card[random_index]
-
-    # Remove the selected number from the list
-    list_card.remove(random_number)
+    random_number = list_card.pop(random_index)
     return random_number
 
+
 def print_stats(computer, user):
-    print(f"computer: [{computer[0]}, *] ")
-    print(f"user : {user} = sum {card_sum(user)} ")
-    print(f"Your wallet fund is ${money}")
+    print(f"Computer: [{computer[0]}, *] ")
+    print(f"User: {user} = Sum {card_sum(user)} ")
+    print(f"Your Wallet Fund: ${money}")
+
+
+def card_sum(cards):
+    return sum(cards)
 
-def card_sum(list):
-    return sum(list)
 
 def skip_computer():
     while card_sum(computer) <= 17:
         computer.append(pick_card())
-    
-
-exit = True
-while money> 0 or not exit:
-    list_card = [11,2,3,4,5,6,7,8,9,10,10,10,10,
-             11,2,3,4,5,6,7,8,9,10,10,10,10,
-             11,2,3,4,5,6,7,8,9,10,10,10,10,
-             11,2,3,4,5,6,7,8,9,10,10,10,10]
+
+
+def display_winner():
+    global money
+    if card_sum(computer) == card_sum(user):
+        print(f"Match Draw: Computer-{computer} and User-{user}")
+        money += pot / 2
+    elif card_sum(computer) > 21 or (21 >= card_sum(user) > card_sum(computer)):
+        print(f"User Wins: ${pot}")
+        money += pot
+    else:
+        print("Computer Wins!")
+
+
+def play_again():
+    return input("Play Again? (y/n): ").lower() == "y"
+
+
+exit_game = False
+
+while money > 0 and not exit_game:
+    list_card = [
+        11,
+        2,
+        3,
+        4,
+        5,
+        6,
+        7,
+        8,
+        9,
+        10,
+        10,
+        10,
+        10,
+        11,
+        2,
+        3,
+        4,
+        5,
+        6,
+        7,
+        8,
+        9,
+        10,
+        10,
+        10,
+        10,
+        11,
+        2,
+        3,
+        4,
+        5,
+        6,
+        7,
+        8,
+        9,
+        10,
+        10,
+        10,
+        10,
+        11,
+        2,
+        3,
+        4,
+        5,
+        6,
+        7,
+        8,
+        9,
+        10,
+        10,
+        10,
+        10,
+    ]
```

Comment:
I like the old way they are presented, its more readable.
![image](https://github.com/yashkite/Code-Practice/assets/32860312/e4e22d4e-b960-471c-9e75-d875cd06cb3e)